.vuepress/components/mermaid.vue

```diff
@@ -1,18 +1,18 @@
 <template>
   <div class="mermaid">
-    <slot></slot>
+    <slot />
   </div>
 </template>
 
 <script>
 export default {
   beforeMount() {
-    import("mermaid/dist/mermaid").then(m => {
+    import("mermaid/dist/mermaid").then((m) => {
       m.initialize({
-        startOnLoad: true
+        startOnLoad: true,
       });
       m.init();
     });
-  }
+  },
```

Comment:
What is this? is it normal? That's the solution you told us about for the infinite loading.

Reply:
No, this is an asynchronous import to get the mermaid package that generates schemas (like the one in the asynchronous guide). It is not new.